packages/sns/src/converters/ledger.converters.ts

```diff
@@ -17,3 +18,11 @@ export const toTransferArg = ({
   from_subaccount: toNullable(from_subaccount),
   created_at_time: toNullable(created_at_time),
 });
+
+export const toCandidAccount = ({
```

Comment:
as we have `toTransferArg` for consistency reason should it be named `toAccountArg` or it's different?

Reply:
`toTransferArg` converts to a type `TransferArg`.

`toCandidAccount` converts to type `Account`. But `toAccount` I found it not clear enough, hence the `Candid` suffix.

To follow exactly the same patter I would have to use `toAccount` or `toCandidTransferArg`. Do you have any preference?

Reply:
I tend to like `toAccount` because it does not mention `Candid`. We convert to Candid but we do so because we call a canister and the did is "just" the description but I'm fine with it, was also just a question, PR was accepted ;)

Reply:
I like to address the comments even though the PR is accepted. Thanks!